tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```
@@ -2789,3 +2793,308 @@ def _(
+            **kwargs,
+        ) -> List[int]:
+            if not is_sm_100f():
+                logger.debug(
+                    f"CuteDSL: SM version {get_sm_version()} is not supported. "
+                    f"CuteDSL FP8 Group Gemm only supports SM 100 family. Skipping all tactics."
+                )
+                return []
+            # [m, k]
+            m, k = inputs[0].shape[0], inputs[0].shape[1]
+            # [group_size, n, k]
+            group_size, n = inputs[1].shape[0], inputs[1].shape[1]
```

Ruff (0.15.0)

[error] Local variable `group_size` is assigned to but never used

Remove assignment to unused variable `group_size`

(F841)